### PR TITLE
Run Playbook through Grammarly

### PR DESCRIPTION
**Pull request by JaninaBodi**
Corrections based on Grammarly's suggestions

### CODE REVIEW

File: src/development-process/README.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ in fact has a negative impact:
   managers can also have another, potentially more substantial negative impact.
   Since a project can only succeed when the different involved stakeholders work
   together and each one's motivations and goals are respected and addressed, it
-  is vital for each stakeholders to understand and appreciate each other's
-  viewpoints. Many projects fail due to disproportionate weight being assigned
+  is vital for each stakeholder to understand and appreciate each other's
+  viewpoints. Many projects fail due to the disproportionate weight being assigned
```
Comment (marcoow) [inline src/development-process/README.md:79]: I think this changes the meaning.

File: src/development-process/README.md [hunk 2]
```diff
@@ -88,19 +88,19 @@ in fact has a negative impact:
...
-execution. It is conceptualised as a rotating role so that every team member
+execution. It is conceptualized as a rotating role so that every team member
 will assume it every once in a while (unless they opt out of the rotation).
 
 ![Rotating Iteration Lead role](../../assets/rotating-itteration-lead-role.svg "The iteration lead role rotates to a different project team member with every iteration")
 
-Making the iteration lead a rotating role ensures that all team members realise
-the perspectives of all project stakeholders instead of getting stuck in their
+Making the iteration lead a rotating role ensures that all team members realize
+the perspectives of all project stakeholders instead of getting stuck on their
```
Comment (marcoow) [inline src/development-process/README.md:103]: I think this changes the meaning.

File: src/development-process/README.md [hunk 3]
```diff
@@ -408,7 +408,7 @@ particular, good pull requests should have:
   _"closes #<issue>"_ in the description
 
 As with issues, all discussions around a particular pull request should happen
-on the pull request's page. If discussions happen in person or via online chat,
+on the page of that pull request. If discussions happen in person or via online chat,
```
Comment (marcoow) [inline src/development-process/README.md:411]: I think this sounds strange and doesn't work.

File: src/development-process/README.md [hunk 4]
```diff
@@ -583,7 +582,7 @@ issue.
 When a design change is ready for review, its author should actively ask another
 team member to review it - ideally via the tools used in the particular project
 if those support it or over online chat etc. if not. Everyone asked for review
-should reply in a timely manner - even if it's to ask for someone else to be
+should reply on time - even if it's to ask for someone else to be
```
Comment (marcoow) [inline src/development-process/README.md:585]: I think this changes the meaning.

File: src/project-procedure/digital-products/README.md [hunk 5]
```diff
@@ -308,10 +307,9 @@ included in the increment (e.g. configuring products of the most successful
 category), which ones will not (e.g. configuring products with little sales
 volume) and which ones will be moved to a later increment in case the team is
 running out of time or budget (e.g. allowing customers to see the status of past
-orders). Introducing these escape hatches allows the project team to make a
-decision between implementing essential versions of more features versus
+orders). Introducing these escape hatches allows the project team to decide between implementing essential versions of more features versus
 covering more non-essential aspects and edge cases for fewer features later in
-the process if time should become scarce.
+the process if the time should become scarce.
```
Comment (marcoow) [inline src/project-procedure/digital-products/README.md:312]: I think this sounds strange.

File: src/development-process/README.md (hunk 1 above)
Comment (JaninaBodi) [reply src/development-process/README.md:79]: You mean the article before 'disproportionate weight'? I wasn't sure so I asked a native language speaker and he said that the article is needed here

File: src/development-process/README.md (hunk 2 above)
Comment (JaninaBodi) [reply src/development-process/README.md:103]: Same thing here, I asked and got the answer that "stuck on" would be the correct expression to use

Comment (marcoow) [reply src/development-process/README.md:103]: ok

File: src/development-process/README.md (hunk 1 above)
Comment (marcoow) [reply src/development-process/README.md:79]: ok

**Review by marcoow — APPROVED** [review]

**Pull request merged**